packages/alias/alias.mjs

```diff
@@ -4,58 +4,63 @@ import { pathToFileURL, URL } from 'node:url';
 
 import _get from 'lodash.get';
 
-
 const projectRoot = pathToFileURL(`${process.cwd()}/`);
 
-const aliases = await readConfigFile('tsconfig.json')
+const aliases = await readConfigFile('tsconfig.json');
 
-if (!aliases) console.warn(
-  'Alias loader was registered but no "paths" were found in tsconfig.json',
-  'This loader will behave as a noop (but you should probably remove it if you aren’t using it).',
-);
+if (!aliases)
+	console.warn(
```

Comment:
This is wrong

Reply:
What is the expected behavior ?

Reply:
It should not move `console.warn(` to a new line

Reply:
Humm I think I this case we cannot do something it's what biome do.
We can still stop using formater but I think the ratio of maintnability time to time when you're not a fan is good. 
And we'll have the same thing with other tools like prettier. Because we're trying to have a human rule, whereas the tool follows a mathematical rule.